client/post-editor/media-modal/index.jsx

```diff
@@ -55,7 +51,10 @@ module.exports = React.createClass( {
 			mediaLibrarySelectedItems: Object.freeze( [] ),
 			onClose: noop,
 			onInsertMedia: noop,
-			labels: Object.freeze( {} )
+			labels: Object.freeze( {} ),
+			setView: () => {},
+			resetView: () => {},
```

Comment:
>should we be using `noop` here?

For consistency in the component, I'd say yes, though if I were to write it again, I'd probably drop the `noop`. Will change.